packages/common/lib/types/StateManager.types.ts

```diff
@@ -4,6 +4,11 @@ export interface CredentialOfferState {
   clientId?: string;
   credentialOffer: CredentialOfferPayloadV1_0_11;
   createdOn: number;
+  'pre-authorized_code': string;
+  preAuthorizedCodeCreatedOn: number;
+  preAuthorizedCodeExpiresIn: number;
+  userPin: number;
```

Comment:
Why are the userPin properties stored if they are already part of the credentialOffer?

Reply:
We have `user_pin_required` inside `GrantUrnIetf` but no `user_pin`. Should I add it to `GrantUrnIetf`?